test_util/aws.py

```diff
@@ -93,7 +93,28 @@ def delete(self):
         logger.info('Starting deletion of CF stack')
         # boto stacks become unusable after deletion (e.g. status/info checks) if name-based
         self.stack = self.boto_wrapper.resource('cloudformation').Stack(self.stack.stack_id)
+        self.empty_and_delete_s3_bucket_from_stack()
```

Comment:
nit: should do this after self.stack.delete(), exhibitor could use the bucket while we're deleting it otherwise.